_posts/2016/2016-05-12-offsite-consulting.md

```diff
@@ -0,0 +1,45 @@
+---
+
+On the 1st of Mai 2016 consulting team began its trip to Waldhotel in Bad Sulzburg. After a six-hour drive, we were greeted by nice staff of the hotel. The Hotel, which is located in the countryside, in a quiet location of the Schwarzwald. A perfect area for us to be clear about what went well and what could be done better in future. 
+
+{% image blog/blog-consulting-offsite-team-1.jpg %}{% endimage %}
+
+The purpose of this particular trip was to talk about opportunities and challenges faced by our team at the time and to envisage strategic plans for the future.
+
+## Team building activities
+
+To weld the team together and go away from hectic workings days with stress and strain, we decided to go hiking through a landscape of exceptional diversity. We were impressed with the grandiose views of the Schwarzwald landscape. 
+
+{% image blog/blog-consulting-offsite-traveling-1.jpg %}{% endimage %}
+
+On the next day after countless meetings we played Viking Chess to reduce stress and to draw fresh inspiration. Viking Chess is an exciting team game from Sweden with a very long tradition. The goal of each team is to overturn the wooden figures of the counterparty by throwing wooden sticks. The king with the red crown, who stands in the middle of the field, must be taken last.
+
+{% image blog/blog-consulting-offsite-traveling-2.jpg %}{% endimage %}
+
+## At the meeting
+
+At the meeting we all worked together on all issues, each topic had his own presenter. The results which were achieved bring many benefits and have a good chance of acceptance by all participants. 
+
+During our work we discussed different topics. First we analyzed the work from the previous two quarters and worked out a plan to reduce some internal expenses in order to focus on work that creates more profit.
+
+{% image blog/blog-consulting-offsite-at-meeting-1.jpg %}{% endimage %}
+
+On the next day we had a very important meeting with our new customer Lexware. We got feedback about our work which was already finished and set milestones for the future collaboration. 
+
+{% image blog/blog-consulting-offsite-at-meeting-2.jpg %}{% endimage %}
+
+Since the work of consulting is to customize the standard epages software according to customer specific requirements, one of our topics was the continuous delivery pipeline. This pipeline should ensure that each time a new version will be released consulting changes are still working and no side effects occur.
+
+## Summary
+
+Despite the fact that Bad Sulzburg was far away and the traveling was very tiring, it was a great place to think out of the box. We had the great opportunity to solve problems, generate new ideas and develop a plan for further work.
+
+From our point of view the offsite meeting is an essential part for every professional team which encourages creativity. An offsite meeting can be especially helpful when you're planning to make strategic decisions. We are looking forward to our next offsite meeting.
```

Comment:
Change to:
From our point of view, a team offsite is an essential part for every professional team which encourages creativity. This can be especially helpful when you plan to make strategic decisions. We look forward to our next offsite meeting.